src/python/pants/rules/core/binary.py

```diff
@@ -31,7 +31,7 @@ class CreatedBinary:
   binary_name: str
 
 
-@console_rule
+@console_rule(name="binary")
```

Comment:
For @console_rule (which requires a `Goal` return type) you should be able to use the `Goal.name` by default.

Reply:
Added this in a subsequent commit which is squished in.